api/src/domain/types.ts

```diff
@@ -50,6 +61,16 @@ export interface SelfRegClient {
   resume: (authID: string) => Promise<SelfRegResponse>;
 }
 
+export type TaxFilingResults = { Content: string; Id: string; Values: string[] }[] | null;
```

Comment:
I think this type does not belong in the domain? It is an api-specific type, and we should never have types directly referencing external APIs in our domain, because changes on their end should never cause changes in our domain (it should only cause changes in our api adapters)

This is another dependency inversion principle. I know you missed that deep-dive when I did it with anju, soma, and faraz a few weeks ago, but I'd be happy to go through that with you to see the why and how of what the DI should look like in our api.

In any case, this should be an easy fix because you don't appear to be actually USING this type anywhere in the domain, just defining it here. It's only needed in the api client anyways, so it should just live there.